rdrf/rdrf/views/form_view.py

```diff
@@ -298,7 +298,7 @@ def _enable_context_creation_after_save(self,
 
     @staticmethod
     def get_form_group_name(patient_model, context):
-        return context.context_form_group.get_default_name(patient_model, context)
+        return context.context_form_group.get_name_from_cde(patient_model, context)
```

Comment:
A function called `get_name_from_cde` that doesn't necessarily get the name from the cde, fun :)

Reply:
Yes, good name!